components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -473,16 +515,23 @@ void AtmosphereOutput::
 set_field_manager (const std::shared_ptr<const fm_type>& field_mgr, const std::string& mode)
 {
   // Sanity checks
-  EKAT_REQUIRE_MSG (mode=="io" || mode=="sim", "Error! Invalid arg to set_field_manager, must be 'io' or 'sim'");
+  EKAT_REQUIRE_MSG (mode =="all" || mode=="int" || mode=="io" || mode=="sim", "Error! Invalid arg to set_field_manager, must be 'io' or 'sim'");
```

Comment:
Suggestion: you start to have quite a few FM here, so maybe a `FMType` enum might be more appropriate than a string.